### PR TITLE
Added param configs from eval to pass to the frontend

### PR DESCRIPTION
**Pull request by yk**

### CODE REVIEW

File: oasst-shared/oasst_shared/schemas/inference.py [hunk 1]
```diff
@@ -8,20 +8,15 @@
...
-DEFAULT_MODEL_NAME = "distilgpt2"
 
 DEFAULT_MODEL_LENGTHS = {
     "_lorem": 1024,
-    "gpt2": 1024,
-    "distilgpt2": 1024,
-    "OpenAssistant/oasst-sft-1-pythia6b": 2048,
     "OpenAssistant/oasst-sft-1-pythia-12b": 2048,
     "OpenAssistant/oasst_sft_llama_7b_mask_1000": 2048,
     "OpenAssistant/oasst_sft_llama_13b_mask_1500": 2048,
```
Comment (olliestanley) [inline oasst-shared/oasst_shared/schemas/inference.py:19]: We can't offer these LLaMa models now either, right?

Comment (yk) [reply oasst-shared/oasst_shared/schemas/inference.py:19]: yes we can, as long as we don't profit monetarily. we can even provide the outputs without providing the actual weights until we figure out the license stuff

Comment (olliestanley) [reply oasst-shared/oasst_shared/schemas/inference.py:19]: Ok - is the reason for removing 30B just that it would require too much compute then?

Comment (yk) [reply oasst-shared/oasst_shared/schemas/inference.py:19]: yes exactly

**Pull request merged**